integration-tests/metrics-expected-changes.patch

```diff
@@ -4343,7 +4382,7 @@
      snapshot.dump(output);
  
 -    assertThat(output.toString()).isEqualTo(String.format("1%n2%n3%n4%n5%n"));
-+    assertThat(output).hasToString(String.format("1%n2%n3%n4%n5%n"));
```

Comment:
This one is interesting 🤔, this doesn't do anything right?

Reply:
It does do something: it replaces `%n` with `System.lineSeparator()`.